client/web/src/insights/hooks/use-delete-insight/use-delete-insight.ts

```diff
@@ -29,9 +29,11 @@ export function useDeleteInsight(props: UseDeleteInsightProps): UseDeleteInsight
     const [error, setError] = useState<ErrorLike | undefined>()
 
     const handleDelete = useCallback(
-        async (insightID: string) => {
+        async (insight: Pick<Insight, 'id' | 'title'>) => {
+            const shouldDelete = window.confirm(`Are you sure you want to delete ${insight.title}?`)
```

Comment:
@AlicjaSuska @felixfbecker @Joelkw Do you have any thoughts on the text to confirmation step here?

Reply:
This could be weird if the title is e.g. "commits" ("Are you sure you want to delete commits?")

```suggestion
            const shouldDelete = window.confirm(`Are you sure you want to delete the insight "${insight.title}"?`)
```